tools/run_tests/pre_build_csharp.bat

```diff
@@ -38,8 +38,21 @@ cd /d %~dp0\..\..
 set NUGET=C:\nuget\nuget.exe
 
 if exist %NUGET% (
+  @rem Restore Grpc packages by packages since Nuget client 3.4.4 doesnt support restore
```

Comment:
Ah, that's really unfortunate. Can't we use some wildcards? Btw, for vsproject/grpc_csharp_ext.sln the per-solution restore works because it's a C++ solution?